functions/snippets/pom.xml

```diff
@@ -52,6 +52,13 @@
       <version>3.4.2</version>
     </dependency>
 
+    <!-- Required for com.example.functions.MakeUpperCaseRTDB -->
```

Comment:
`MakeUpperCaseRTDB` implies Realtime Database (RTDB) but I see Firestore here.  Not sure if just the comment is wrong or something else needs to change.

Reply:
Ah - this is out-of-date. Fixed.